From 17d1dd1e18da6200a5062d7d586458b6140d36a3 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Fri, 16 Mar 2012 11:40:02 +0800
Subject: [PATCH 1185/2508] ENGR00177046: Add the platform dependency for PXP
 in Kconfig

If there's no dependency, build will be broken when do
`make ARCH=arm CROSS_COMPILE=<cross-compiler path> allmodconfig'
`make'
because PXP module will be turned on. This patch fixed it.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 drivers/dma/Kconfig |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/dma/Kconfig b/drivers/dma/Kconfig
index 7d6b900..fda1193 100644
--- a/drivers/dma/Kconfig
+++ b/drivers/dma/Kconfig
@@ -143,10 +143,12 @@ config MX3_IPU_IRQS
 
 config MXC_PXP
     bool "MXC PxP support"
+    depends on ARCH_MX50
     select DMA_ENGINE
 
 config MXC_PXP_V2
     bool "MXC PxP V2 support"
+    depends on ARCH_MX6
     select DMA_ENGINE
 
 config MXC_PXP_CLIENT_DEVICE
-- 
1.7.7.4

